---
fontsize: 12pt
geometry: margin=0.50in
subparagraph: yes

title: "Analysis of integration site distributions and clonal abundances for gene therapy correction of cystinosis (Groups 29-30)"
date:  "June 2019"
author: "John K. Everett, Ph.D. and Frederic Bushman, Ph.D."
header-includes:
    - \usepackage{caption}
output:
  pdf_document:
    toc: true
    keep_tex: no
    latex_engine: xelatex
---
\captionsetup[table]{labelformat=empty}

```{r, echo=FALSE, message=FALSE, warning=FALSE}
load('project.group3.RData')
library(tidyverse)
library(gt23)
library(xtable)
library(RColorBrewer)
library(gridExtra)
library(knitr)
library(kableExtra)
```

\newpage

# Summary of results

The goal of this analysis is to investigate the integration profile of a gene therapy vector for the correction of cystinosis 
in mouse subjects and assess potential clonal expansions. The list of mouse oncogenes was compiled from the retroviral tagged 
cancer gene database (RTCGD)^1^ using an inclusion threshold of three or more incidents where the mouse oncogene list comprises 
`r mouseGenomePercentOnco`% of all mouse genes. The frequency of integration near oncogenes was 
generally less than that of mice in a previously published $\beta$-thalassemia mouse trial from which no adverse events have 
been reported ^2^.  The code base for this analysis is available online 
([link](https://github.com/everettJK/project.geneTherapy.ucsdMouseCystinosis)).  

\vspace{0.25cm}

Twenty-eight mice were tested by serial transfer of vector-treated bone marrow. In both donor and recipient samples, there was no overall increase in integration sites near annotated murine cancer-associated genes, thus providing no evidence for genotoxicity.  Some cells expanded more than others, but analysis of cells showing the most pronounced expansion showed no enrichment in integration sites in or near cancer-associated genes. In a number of cases, a single cell appears to have mainly populated a test mice. These clones, for the most part, appear to multi-integrated cells which appear as multiple nearly equal size bars within the relative abundance plots within this report.

\newpage

# Mouse samples studied

Integration sites were detected in `r length(unique(intSites$GTSP))` samples from mouse subjects (Tables 1 & S1).

\vspace{0.1cm}

```{r, echo=FALSE, eval=TRUE}
names(summaryTable) <- c('Organism', 'Number of samples', 'Number of reads', 'Number of inferred cells', 'Number of integration sites')
kable(summaryTable,  format = "latex", align='c', caption = 'Table 1. Overview of data collection.', booktabs = T) %>%
  kable_styling(latex_options = c("hold_position"))
```

\vspace{0.5cm}

# Subject reports
Subject specific reports for all subjects are available via an online archive ([link](http://bushmanlab.org/data/export/projects/gtVISA/CYS/)).  

\vspace{0.5cm}

# UCSC browser exploration
UCSC browser sessions pre-loaded with the integration sites identified in this analysis are available via this  
([link](http://genome.ucsc.edu/cgi-bin/hgTracks?org=mouse&db=mm9&hgt.customText=http://microb120.med.upenn.edu/UCSC/cherqui/UCSC_CYS_mouse.group2.ucsc)). 
Integration sites are shown as blue (positive orientation integration) and red (reverse orientation integration) tick marks. For each integration site, 
a second track provides the maximum clonal abundance. Entering gene names into the search bar will direct the browser to specific genes. 

\newpage

# Description of analysis techniques  

We investigate effects of integration on cell growth using the following criteria:
Integration Frequency is the frequency at which unique integration sites are observed in or near a given gene.
Clonal Abundance is determined by quantifying the number of sites of linker ligation associated with each unique integration
site. This samples the number of DNA chains at the start of the experiment allowing clonal expansion to be quantified^4^.  

Relative clonal Abundance is determined per sample and is the percentage of identified cells attributed to a given clone.
Integration sites and the clones harboring them are sampled from a larger population. It would be rare for all integration
sites in a sample to be represented in the sequence data.  
  
For this analysis, four technical replicates of each delivered sample were prepared, sequenced and analyzed with the 
INSPIIRED integration site analysis pipeline (v1.2)^4^.

\newpage

# Comparisons to previous trials

## Integration events near oncogenes in mouse subjects  

\vspace{0.1cm}

In order to determine if the experimental vector has a higher propensity of integrating near suspected oncogene in mice than
previously employed vectors, the frequency of integration near oncogenes was compared to a previously published mouse trial^2^ 
which used a comparable lentiviral vector to correct $\beta$-thalassemia.  The frequency of integration events near onco genes 
in *all transfected mice* was less than the mean frequency of integration events near oncogenes in the published trial 
(Figure 1 [experimental subjects: gray, previous $\beta$-thalassemia trial: blue]).  

\vspace{1.0cm}

Figure 1. Comparison of frequencies of integration events near oncogenes.  

\vspace{0.5cm}

```{r fig1, echo = FALSE, fig.height=4}
PMC3129560vsMouseSubjects
```

\newpage

# Relative abundances of mouse subject samples

The sample relative abundance plots below (Figure 2) show the most abundant 25 clones in each sample as colored 
bars while less abundant clones were relegated to a single low abundance bar shown in gray. 

\vspace{0.5cm}

Figure 2.  

\vspace{0.5cm}

```{r, echo=FALSE, eval=TRUE, fig.height=7, Figurewidth=8, cache=FALSE}
grid.arrange(grobs=mouseRelAbundPlots[1:16], ncol=4)
```

\newpage

Figure 2 (continued).  

\vspace{0.50cm}

```{r, echo=FALSE, eval=TRUE, fig.height=6, Figurewidth=8, cache=FALSE}
grid.arrange(grobs=mouseRelAbundPlots[17:length(mouseRelAbundPlots)], ncol=4)
```

\newpage

# Expanded clones  

Table 2 below lists clones with relative clonal abundances $\geq$ 20%. The estimated number of cells harboring each integration 
(Abundance) is shown for context.  

\vspace{1.0cm}

Table 2.  

```{r table2, echo=FALSE, eval=TRUE, cache=FALSE, results='asis'}
names(abundantClones20) <- c('Subject', 'Organism', 'Time point', 'Cell type', 'Position', 'Relative abundance', 'Abundance', 'Nearest gene')
kable(abundantClones20, format = "latex", linesep = "", booktabs = T) %>% kable_styling(latex_options = c("striped", "scale_down"))
```

\newpage

# Mapping of integration site positions  
Integration events were observerd across all mouse subject chromosomes (Figure 3). 

\vspace{1.0cm}

Figure 3.  

\vspace{0.25cm}

```{r fig3a, echo = FALSE, fig.height=6, dev='png', dpi=150, cache=FALSE}
mouseIntSiteMap
```


\newpage

**Analyst**    
```{r echo=FALSE, Figurewidth=1}
knitr::include_graphics('./data/Everett_signature.png')
```
   
John K. Everett, Ph.D.

\vspace{0.5cm}

**Laboratory director**    
```{r echo=FALSE, Figurewidth=1}
knitr::include_graphics('./data/Bushman_signature.png')
```
   
Frederic D. Bushman, Ph.D.

\vspace{2.0cm}

# References
  
1. RTCGD: retroviral tagged cancer gene database. Akagi K, Suzuki T, Stephens RM, Jenkins NA, Copeland NG. 
Nucleic Acids Res. 2004 Jan 1;32(Database issue):D523-7.

\vspace{0.1cm}

2. Distribution of Lentiviral Vector Integration Sites in Mice Following Therapeutic Gene Transfer to Treat $\beta$-thalassemia. 
Ronen K, Negre O, Roth S, Colomb C, Malani N, Denaro M, Brady T, Fusil F, Gillet-Legrand B, Hehir K, Beuzard Y, Leboulch P, Down JD, Payen E, Bushman FD. 
Mol Ther. 2011 Jul;19(7):1273-86.

\vspace{0.1cm}

3. Estimating abundances of retroviral insertion sites from DNA fragment length data. Berry CC, Gillet NA, Melamed A, Gormley N, Bangham CR, Bushman FD. 
Bioinformatics. 2012 Mar 15;28(6):755-62.

\vspace{0.1cm}

4. INSPIIRED: A Pipeline for Quantitative Analysis of Sites of New DNA Integration in Cellular Genomes. Sherman E, 
Nobles C, Berry CC, Six E, Wu Y, Dryga A, Malani N, Male F, Reddy S, Bailey A, Bittinger K, Everett JK, Caccavelli L, 
Drake MJ, Bates P, Hacein-Bey-Abina S, Cavazzana M, Bushman FD. Mol Ther Methods Clin Dev. 2016 Dec 18;4:39-49. 

\newpage

# Supplementary tables and figures

## Numbers of inferred cells and integration sites identified in provided samples

Table S1.  

```{r tableS1, echo=FALSE, eval=TRUE, cache=FALSE}
names(sampleSummary) <- c('Organism', 'GTSP', 'Subject', 'Cell type', 'VCN', 'Time point', 'Number inferred cells', 'Number of intSites')
kable(sampleSummary, format = "latex", linesep = "", booktabs = T) %>% kable_styling(latex_options = c("striped", "scale_down"))
```

\newpage

## Sequencing depth  
Identified integration site are shown as colored squares that are positioned by the number of reads leading to their identification.  

Figure S2.  
  
```{r FigS3cont, echo=FALSE, eval=TRUE, fig.height=8, cache=FALSE, dev='png', dpi=150}
intSiteReadsPlot
```

